runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -304,6 +304,15 @@ void setupSwaps()
 		swap("teleport menu", "draynor village", config::swapJewelleryBox);
 		swap("teleport menu", "al kharid", config::swapJewelleryBox);
 
+		Arrays.asList(
+				"grand exchange", "varrock", "lumbridge grave'", "draynor manor", "battlefront", "mind altar",
+				"lumbridge", "falador", "salve graveyard", "camelot", "seers' village", "fenken' castle",
+				"east ardougne", "watchtower", "yanille", "senntisten", "west ardougne", "marim", "harmony island",
+				"kharyll", "lunar isle", "kourend castle", "the forgotten cemetery", "waterbirth island", "barrows",
+				"carrallangar", "fishing guild", "catherby", "annakarl", "ape atoll dungeon", "ghorrok",
+				"troll stronghold", "weiss"
+			).forEach(location -> swap(location, "teleport menu", config::swapPortalNexus));
```

Comment:
Reduce this indentation by one level.
```suggestion
			"grand exchange", "varrock", "lumbridge grave'", "draynor manor", "battlefront", "mind altar",
			"lumbridge", "falador", "salve graveyard", "camelot", "seers' village", "fenken' castle",
			"east ardougne", "watchtower", "yanille", "senntisten", "west ardougne", "marim", "harmony island",
			"kharyll", "lunar isle", "kourend castle", "the forgotten cemetery", "waterbirth island", "barrows",
			"carrallangar", "fishing guild", "catherby", "annakarl", "ape atoll dungeon", "ghorrok",
			"troll stronghold", "weiss"
		).forEach(location -> swap(location, "teleport menu", config::swapPortalNexus));
```